### ISSUE TITLE
Allow queries exceeding the 20,000 entries limit of the APIs

### ISSUE THREAD

**Issue by PascalIrz** [inrae/hubeau#32]
It would be great improvement for the user!

**Comment by cedricbriandgithub** [comment]
Agreed !
This comes from the API itself when trying to override

> Message: Error 400 on query: https://hubeau.eaufrance.fr/api/v0/indicateurs_services/communes?page=11&size=2000
> Error on parameters:
> page please refine your query. The multiplication of `page` * `size` parameters can not be more than 20000

**Comment by PascalIrz** [comment]
So far I proceed as follows in 2 steps, so that each call to the get_xx_xx() function does not exceed 20000 entries:

`library(hubeau)

stations <- get_niveaux_nappes_stations(
  code_departement = '79')

piezo <- map_df(
  .x = stations$code_bss,
  .f = function(x)
    get_niveaux_nappes_chroniques(code_bss = x,
                                  date_debut_mesure = "2015-01-01")
)`

**Comment by PascalIrz** [comment]
Sure, it is just a workaround. @DDorch may have better ideas for a generic solution.

**Comment by DDorch** [comment]
I agree with @cedricbriandgithub it's impossible to guess how many records would return a query whatever the way you cut it into subqueries. Event a fine treatment for each API could failli over time...

The best thing could be to add a documentation on how to processus in case of a query returning more thanks 20000 records.
